docs/yaml/functions/install_symlink.yaml

```diff
@@ -0,0 +1,33 @@
+name: install_symlink
+returns: void
+description: |
+  Installs a symbolic link to `pointing_to` target under install_dir.
+
+varargs:
+  name: link_name
+  type: str
+  description: |
+    Name of the created link under `install_dir`.
+    It cannot contain path separators. Those should go in `install_dir`.
+
+kwargs:
+  pointing_to:
+    type: str
+    description: |
+      Target to point the link to.
+      Can be absolute or relative and that will be respected when creating the link.
+
+  install_dir:
+    type: str
+    description: |
+      The absolute or relative path to the installation directory for the links.
+      If this is a relative path, it is assumed to be relative to the prefix.
+
+      If omitted, the directory defaults to `{datadir}/{projectname}`.
```

Comment:
I'm trying to imagine how useful this default would actually be... There is nothing particular about symlinks that I can imagine, which would actually benefit from this that I can think of.

I think it would be clearer to just make the install_dir required. If someone comes up with a convincing use case for adding a default, we can always change that later, but it's difficult to drop support once we add it.